spec-main/api-context-bridge-spec.ts

```diff
@@ -472,6 +472,39 @@ describe('contextBridge', () => {
         expect(threw).to.equal(true);
       });
 
+      it('should copy thrown errors into the other context', async () => {
+        await makeBindingWindow(() => {
+          contextBridge.exposeInMainWorld('example', {
+            throwNormal: () => {
+              throw new Error('whoops');
+            },
+            throwWeird: () => {
+              throw 'this is no error...'; // eslint-disable-line no-throw-literal
+            },
+            throwNotClonable: () => {
+              return Object(Symbol('foo'));
+            },
+            argumentConvert: () => {}
+          });
+        });
+        const result = await callWithBindings((root: any) => {
+          const getError = (fn: Function) => {
+            try {
+              fn();
+            } catch (e) {
+              return e;
+            }
+            return null;
+          };
+          const normalIsError = Object.getPrototypeOf(getError(() => root.example.throwNormal())) === Error.prototype;
```

Comment:
```suggestion
          const normalIsError = Object.getPrototypeOf(getError(root.example.throwNormal)) === Error.prototype;
```

Reply:
Not sure it matters in this particular case so I could change this but I'm not a huge fan of passing methods by property access as it rebinds / loses the `this` binding in many cases and can do so silently.  It's normally safer / more consistent to pass a lamda style function.